rabit/rabit/Goal/CertPhotoCamera/CertPhotoCameraViewModel.swift

```diff
@@ -36,9 +36,9 @@ private extension CertPhotoCameraViewModel {
             .disposed(by: disposeBag)
         
         nextButtonTouched
-            .withLatestFrom(previewPhotoData)
-            .map { Photo(categoryTitle: goal.category, goalTitle: goal.title, imageData: $0, date: Date(), color: "", style: .none) }
-            .map { BehaviorRelay(value: $0) }
+            .withLatestFrom(previewPhotoData) { (goal.category, goal.title, $1) }
+            .map(Photo.init)
+            .map(BehaviorRelay.init)
```

Comment:
```suggestion
            .map(BehaviorRelay(value: Photo.init))
```
혹시 가능하면 이렇게 줄이는게 보기 편할 것 같아요!